controlsfx/src/main/java/org/controlsfx/control/textfield/ClearEvent.java

```diff
@@ -0,0 +1,19 @@
+package org.controlsfx.control.textfield;
+
+import javafx.event.EventType;
+import javafx.scene.input.InputEvent;
+
+/**
+ * An event which indicates that a clear icon has been pressed in a {@link TextFields#createClearableTextField()} or {@link TextFields#createClearablePasswordField()}.
+ *
+ * This event is generated after cleaning a text for the TextField
```

Comment:
Maybe use `clearing` instead of  `cleaning` ? What do you think?

Reply:
done